docs/strategies/fallback.md

```diff
@@ -65,6 +65,45 @@ new ResiliencePipelineBuilder<UserAvatar>()
 | `FallbackAction` | `Null`, **Required**                                                       | Fallback action to be executed.                                                             |
 | `OnFallback`     | `null`                                                                     | Event that is raised when fallback happens.                                                 |
 
+## Diagrams
+
+### Happy path sequence diagram
+
+```mermaid
+%%{init: {'theme':'dark'}}%%
```

Comment:
This looks quite bad if you don't use the dark theme: https://github.com/App-vNext/Polly/pull/1701#discussion_r1361723745

Reply:
On the [mermaid documentation site](https://mermaid.js.org/syntax/sequenceDiagram.html#notes) the diagrams are dynamically switching between `default` and `dark` theme depending on the selected appearance mode. I haven't figured out yet how they do that so, I will spend some time to dig deeper.

Reply:
How about we just don't do anything and raise an issue in DocFx repo?

My thinking is that default behavior should handle this automatically.

Reply:
I would agree that it should _just work_.

If we can't make it look good for both light and dark, then we should use whichever combination looks _less worse_.

How does the default/light diagram look in dark mode?

Reply:
The defaults for white theme:

![image](https://github.com/App-vNext/Polly/assets/103487740/a6c28548-6c6d-4798-b9e4-e9eff87dd4aa)

Dark theme:
![image](https://github.com/App-vNext/Polly/assets/103487740/ebd8137b-0e2b-4b51-9b75-b81d8360f2c9)

Reply:
I've found how [they do it](https://github.com/mermaid-js/mermaid/blob/33a5fc459d852e2af42c749ac982be38cc9802e8/packages/mermaid/src/docs/.vitepress/theme/Mermaid.vue#L59C12-L59C24). :)
```typescript
const renderChart = async () => {
  const hasDarkClass = document.documentElement.classList.contains('dark');
  const mermaidConfig = {
     ...   
    theme: hasDarkClass ? 'dark' : 'default',
  };
  ...
}
```

I don't we can do similar via the docfx config :(

Reply:
Is there a way to add a custom module/script to apply a global config or something when the page is being rendered in the client? [Docs](https://dotnet.github.io/docfx/tutorial/intro_template.html#module)

If not, would be good to create an issue in the docfx repo if there isn't one already requesting this feature.

Reply:
Shall we add a `note` to the diagrams sections something like: 

 _Please prefer dark appearance mode for better legibility ... it is a temporarily solution_

?

Reply:
Let's avoid a comment - otherwise if we have lots of diagrams I think it's just going to duplicate all over the place.